### PR TITLE
feat: add simple validation to adjusted grade field

### PR DESCRIPTION
**Pull request by ihor-romaniuk**
**TL;DR** - added simple validation for the "Adjusted grade" field in the "Edit Grades" popup:
- enter only a numeric value
- setting the minimum value
- checking the correctness of the entered value to possibleGrade

FYI: @openedx/content-aurora

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @ihor-romaniuk!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by natabene** [comment]
@ihor-romaniuk Thank you for the contribution. Is this ready for our review?

**Comment by ihor-romaniuk** [comment]
> @ihor-romaniuk Thank you for the contribution. Is this ready for our review?

Yes, everything is ready for review.

**Comment by ProductRyan** [comment]
@mattcarter - any concerns about this change from your POV?

**Comment by ProductRyan** [comment]
Creating a JIRA ticket for internal tracking:
https://2u-internal.atlassian.net/browse/AU-944

File: src/components/GradesView/EditModal/OverrideTable/AdjustedGradeInput.jsx [hunk 1]
```diff
@@ -20,15 +20,28 @@ export class AdjustedGradeInput extends React.Component {
   }
 
   onChange = ({ target }) => {
-    this.props.setModalState({ adjustedGradeValue: target.value });
+    let adjustedGradeValue;
+    switch (true) {
```
Comment (muselesscreator) [inline src/components/GradesView/EditModal/OverrideTable/AdjustedGradeInput.jsx:24]: why set min/max AND re-enforce onChange?

Comment (ihor-romaniuk) [reply src/components/GradesView/EditModal/OverrideTable/AdjustedGradeInput.jsx:24]: Input attributes min/max validate value which was changed by input number controls but it doesn't validate key entering.
So an additional validation in the `onChange` method was created for validation value from the keyboard.

Comment (ihor-romaniuk) [reply src/components/GradesView/EditModal/OverrideTable/AdjustedGradeInput.jsx:24]: Also, the package-lock.json file was updated because of PR conflicts.

**Review by leangseu-edx — CHANGES_REQUESTED** [review]
Your `package-lock.json` is using the wrong version. Please use `node-16`

**Comment by mphilbrick211** [comment]
Hi @ihor-romaniuk! Just flagging that some changes were requested on this pull request. Please let us know if you have any questions.

**Comment by ihor-romaniuk** [comment]
> Your `package-lock.json` is using the wrong version. Please use `node-16`

The package-lock.json was updated with using node-16.

**Comment by ihor-romaniuk** [comment]
> Hi @ihor-romaniuk! Just flagging that some changes were requested on this pull request. Please let us know if you have any questions.

Hi. All comments were solved, and it's ready for review.

**Comment by leangseu-edx** [comment]
closed this and reopen https://github.com/openedx/frontend-app-gradebook/pull/263 because I couldn't push change directly to this branch.

**Comment by openedx-webhooks** [comment]
@ihor-romaniuk Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-gradebook&entry.867055334=https://github.com/openedx/frontend-app-gradebook/pull/263&entry.1484655318=https://github.com/ihor-romaniuk&entry.752974735=2022-09-12+12:56&entry.1917517419=2023-02-06+14:23&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**